views/image-body.html

```diff
@@ -19,8 +19,8 @@
     <dd ng-if="!labels['build-date'] && !layers[0].v1Compatibility.created && image.dockerImageMetadata.Created" title="{{image.dockerImageMetadata.Created}}">{{image.dockerImageMetadata.Created | dateRelative}}</dd>
     <dt translate>Digest</dt>
     <dd><tt>{{ image.metadata.name }}</tt></dd>
-    <dt translate>Identifier</dt>
-    <dd><tt>{{ config.Image }}</tt></dd>
+    <dt ng-if="config.Image" translate>Identifier</dt>
```

Comment:
yeah, was thinking about it, but we use the same (dup.) logic on other places in this file so want sure which one to use. 
Will update